README.md

```diff
@@ -29,3 +31,22 @@ setRedListToken('<your-token>')
 ```
 
 Be sure to reference the [Red List API docs](http://apiv3.iucnredlist.org/api/v3/docs)
+
+## Threat - Species threat information
+* [Threat.fetch()](#threat-fetch)
+
+<h3 id='threat-fetch'>Threat.fetch(options)
+
+Retrieves a list of threats by either species `name` or species `id`
+
+##### `options` (required) - **[Object]**
+* `name` - **[String]** Name of species to filter threats by
```

Comment:
Should have a note on how this works. "If `name` is not provided, `id` must be provided."